### DISCUSSION TITLE
Bring back loopAdditionalSlides please

### DISCUSSION THREAD

**Discussion by MarioMarusic** [nolimits4web/swiper#6755] [General]
I use swiper on all my sites and on lot of them I have a looped carousel where active slide is 100% opacity but all others are semi transparent, so you can see them animate in. In v9 though now I have a glitch on every loop start and end, because there are not enough slides visible, even with new "loopedSlides" option. So, please bring back loopAdditionalSlides somehow. Thanks.

**Comment by myh110** [comment]
I also encountered the same problem
